spec2vec/Spec2Vec.py

```diff
@@ -1,11 +1,12 @@
 from typing import List
 from typing import Union
 import numpy
-import scipy
```

Comment:
You dont import scipy anywhere else, so scipy should be removed as a dependency in conda/*

Reply:
Missed than one, will be removed of course.

Reply:
Oh, just saw that this went wrong due to a (wrongly sovled) merge conflict, so all those ``scipy`` mentions in the environment should of course be gone by now. I'll remove them.